.azure-pipelines/variables.yml

```diff
@@ -3,7 +3,7 @@ variables:
   - name: versionPrefix
     value: 19.0.0
   - name: versionSuffix
-    value: 'preview.4'
+    value: 'preview.5'
```

Comment:
You don't need to bump prerelease version since `preview.4` wasn't release. This PR is still part of the upcoming `preview.4`

Reply:
That's the only way to keep our CI artifact feed reasonable. Preview.4 is already reviewed and merged to develop branch and someone coud even use it for some reason. If we start pushing these changes as preview.4 there will be no way to know how stable is this build/from which branch it comes. I think it helps to separate these changes and indicate stages of our work. It's not that we can run out of these prevew numbers

Reply:
The CI artifact feed is only for testing. It's like an internal private API that might and will break. That's ok. People shouldn't use development artifacts in production.